cassandra-4.0/Dockerfile

```diff
@@ -1,6 +1,15 @@
 FROM openjdk:8u252-slim
 
+ARG STARGATE_VERSION=v0.0.5
+ARG PERSISTENCE_VERSION=v0.0.1
 
+RUN apt update && apt install curl unzip iproute2 libaio1 -y
+
+RUN curl -sL https://github.com/stargate/stargate/releases/download/${STARGATE_VERSION}/stargate-jars.zip > stargate-jars.zip
+RUN curl -sL https://github.com/stargate/persistence-extensions/releases/download/${PERSISTENCE_VERSION}/stargate-jars.zip > persistence-jars.zip
```

Comment:
Aren't we going to have an explosion is image variants when api extensions come into play as well? Would it be possible to select non-core components as build time options to avoid having a separate docker file for each configuration?

Reply:
We're not going to have an image for every combination, only popular ones and then an uber image. The problem with _only_ having an image with everything is the size. Pulling a multi-gigabyte image in k8s is painful.